### ISSUE TITLE
Work with DNAStack WES endpoint to finalize WES CLI

### ISSUE THREAD

**Issue by cricketsloan** [dockstore/dockstore#2207]
Work with Patrick Magee <[patrick@dnastack.com](mailto:patrick@dnastack.com)> (CEO Marc Fiume <marc@dnastack.com>) and DNAStack WES endpoint to test Dockstore WES CLI and debug and add special features for support of DNAStack WES endpoint

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-576)
┆Issue Type: Story
┆Fix Versions: Dockstore 1.7
┆Issue Number: DOCK-576
┆Epic: WES API support

**Comment by cricketsloan** [comment]
➤ Walt Shands commented:

Latest from Patrick Magee: “At the moment, the WES implementation does not cover 100% of the feature set of WES, namely the ability to specify a workflow content directly in the request. As of now, all workflows need to be created within DNAstack and the only workfow_url supported is the one pointing to a workflow or a workflow version within your DNAstack project. Additionally, we are waiting on some upcoming changes that are finished but have yet to be deployed in production, which modify the way inputs are defined to make it easier for users to specify the payload, as well as adding support for files not currently in DNAstack”

Because of the state of the DNAStack endpoint we will postpone working with it

**Comment by denis-yuen** [comment]
Re-open when available
